conanfile.py

```diff
@@ -53,4 +54,15 @@ def build(self):
             self.cmake.test()
 
     def package(self):
-        self.cmake.install()
+        self.copy(pattern="*", dst="include", src="include")
+        self.copy("LICENSE.txt", dst="licenses", src=".")
+
+    def package_id(self):
+        self.info.header_only()
+
+    def package_info(self):
```

Comment:
Fill in package info so that Conan can generate the CMake module for the package.